picard/browser/filelookup.py

```diff
@@ -115,8 +117,8 @@ def tag_lookup(self, artist, release, track, trackNum, duration, filename):
     def collection_lookup(self, userid):
         return self._build_launch('/user/%s/collections' % userid)
 
-    def _search(self, type_, query, adv=False):
-        if self.mbid_lookup(query, type_):
+    def search_entity(self, type_, query, adv=False, mbid_matched_callback=None):
```

Comment:
PR looks good to me, although I wonder if it would be nice to have a pyqtsignal for this instead.
Not sure.

Reply:
> PR looks good to me, although I wonder if it would be nice to have a pyqtsignal for mbid_match instead.
> Not sure.

Not a bad idea, but FileLookup doesn't inherit from QObject. I'll test this approach, dunno if it is worth.